content/md/en/docs/tutorials/light-clients/browser-extension.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Download the browser extension
+description: Why and how to download the Substrate Connect browser extension.
+---
+
+Substrate connect provides a way to interact with [substrate](https://substrate.dev/)
+based blockchains in the browser without using an RPC server. Substrate connect
+uses a [smoldot](https://github.com/paritytech/smoldot/) WASM light client to
+securely connect to the blockchain network without relying on specific 3rd parties.
```

Comment:
we use a "one sentence per line" rule in these doc's md files - no line brakes on char/word count. Please update  to this?
```
This is a line.
This is a really long line, maybe too long some would say, even the general people have a hard time thinking in  stances this long!
```

Reply:
https://github.com/substrate-developer-hub/substrate-docs/pull/1122/commits/1454831ed5bc7cb182dd97f868d7430812c77071